hs_core/views/user_rest_api.py

```diff
@@ -21,10 +34,20 @@ def get(self, request):
             user_info['last_name'] = request.user.last_name
 
         user_profile = UserProfile.objects.filter(user=request.user).first()
-        if user_profile.title:
-            user_info['title'] = user_profile.title
-        if user_profile.organization:
-            user_info['organization'] = user_profile.organization
+        if user_profile.middle_name:
+            user_info['middle_name'] = user_profile.middle_name
+        if user_profile.state:
+            user_info['state'] = user_profile.state
+        if user_profile.zipcode:
+            user_info['zipcode'] = user_profile.zipcode
+        if user_profile.country:
+            user_info['country'] = user_profile.country
+        if user_profile.date_of_birth:
+            user_info['date_of_birth'] = user_profile.date_of_birth
+        if user_profile.last_four_ss:
+            user_info['last_four_ss'] = user_profile.last_four_ss
+        if user_profile.phone_1:
+            user_info['phone_1'] = user_profile.phone_1
```

Comment:
I hate to see a bunch of duplicate if blocks like this but I totally understand the need for them. You can use `getattr()` with a default instead:

```
user_info_fields = ['middle_name', 'state', etc etc]
for field in user_info_fields:
    user_info[field] = getattr(request.user.userprofile, field, '')
```

Reply:
(then you wouldn't need to supply the defaults in the definition of `user_info` up above either...)